src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -363,7 +367,13 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
     try {
       const ibcMsg = {
         typeUrl: params.msgTypeUrl, // '/ibc.applications.transfer.v1.MsgTransfer',
-        value: params.msg,
+        value: {
+          ...params.msg,
+          timeoutTimestamp: params.msg.timeoutTimestamp
+            // Squid returns timeoutTimestamp as Long, but the signer expects BigInt
+            ? BigInt(Long.fromValue(params.msg.timeoutTimestamp).toString())
```

Comment:
slightly confused by the comment. If squid returns the `timeoutTimestamp` as Long, why do we use `Long.fromValue`? Is it guaranteed that the `timeoutTimestamp` will always be an int?

Reply:
`timeoutTimestamp` technically is just an object (e.g `{ low: 1000, high: 0, unsigned: false }` after being `JSON.parsed`, so we have to convert it to `Long` ourselves before calling `toString` to get the value. 
I think this was done by cosmjs-type encoder before the upgrade.